.babelrc

```diff
@@ -6,16 +6,52 @@
     "transform-class-properties",
     "array-includes",
     [
-        "transform-runtime", {
-            "polyfill": false,
-            "regenerator": true
-        }
+      "transform-runtime", {
+        "polyfill": false,
+        "regenerator": true
+      }
     ]
   ],
   "presets": [
-	  "flow",
-	  "es2016",
-	  "es2017",
-	  "react"
-  ]
+    "flow",
+    "react",
+    ["env", {
+      "targets": {
+        "node": "current"
```

Comment:
I've had babel default to transpiling all source to the current version of node. This should decrease the amount of unnecessary work that's happening.